enterprise/views.py

```diff
@@ -758,9 +759,12 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
         except (AttributeError, ValueError):
             course_effort = ''
         else:
-            course_effort = self.context_data['effort_hours_text'].format(
-                hours=effort_hours
-            )
+            course_effort = ungettext(
+                '{hours} hour per week, per course',
+                '{hours} hours per week, per course',
+                effort_hours,
+            ).format(hours=effort_hours)
```

Comment:
Using `ungettext` was a smart move!

Nit: Do you think it'd be possible, though, to stick this function unformatted into `context_data` under the same key we used before? We can then re-use it here (and maybe elsewhere) as it was being used previously exactly the same way.